### ISSUE TITLE
LP IFI 2024 > Desktop > introduction

### ISSUE THREAD

**Issue by Sahani14** [adfinitas-app/curie-ifi20#6]
Problème d'affichage ici avec l'aplat blanc qui est visible sous l'aplat violet/transparent : 
<img width="1437" alt="image" src="https://github.com/adfinitas-app/curie-ifi20/assets/150721779/40a32c0d-eddf-4082-bac4-d8f102ada7cb">

Il faudrait que l'aplat blanc démarre après l'aplat violet

**Comment by Sahani14** [comment]
As-tu une question sur ce retour ?

**Comment by jean-baptisteAZAN** [comment]
non, juste niveau technique je bloque un peu, je vais en parler a max, je le met en stand by pour l'instant
